diff --git a/dist/Repo.js b/dist/Repo.js
index 7104e0130587b239a9b835e6fad1c90dd6feab13..a0a661f685ce3e354c831c966be023381e894cd0 100644
--- a/dist/Repo.js
+++ b/dist/Repo.js
@@ -577,6 +577,7 @@ export class Repo extends EventEmitter {
             handle.update(doc => {
                 return Automerge.loadIncremental(doc, binary);
             });
+            this.#registerHandleWithSubsystems(handle);
             return handle;
         }
         else {
