diff --git a/lib/browser.js b/lib/browser.js
index 0561f9a9662f7606dcf96b1929bee1223c30858c..cf68959a3499143b03765640e97c01e7483b69f9 100644
--- a/lib/browser.js
+++ b/lib/browser.js
@@ -1,18 +1,20 @@
 'use strict';
 
-exports.MediaStream = window.MediaStream;
-exports.MediaStreamTrack = window.MediaStreamTrack;
-exports.RTCDataChannel = window.RTCDataChannel;
-exports.RTCDataChannelEvent = window.RTCDataChannelEvent;
-exports.RTCDtlsTransport = window.RTCDtlsTransport;
-exports.RTCIceCandidate = window.RTCIceCandidate;
-exports.RTCIceTransport = window.RTCIceTransport;
-exports.RTCPeerConnection = window.RTCPeerConnection;
-exports.RTCPeerConnectionIceEvent = window.RTCPeerConnectionIceEvent;
-exports.RTCRtpReceiver = window.RTCRtpReceiver;
-exports.RTCRtpSender = window.RTCRtpSender;
-exports.RTCRtpTransceiver = window.RTCRtpTransceiver;
-exports.RTCSctpTransport = window.RTCSctpTransport;
-exports.RTCSessionDescription = window.RTCSessionDescription;
-exports.getUserMedia = window.getUserMedia;
-exports.mediaDevices = navigator.mediaDevices;
+if (typeof window !== 'undefined') {
+  exports.MediaStream = window.MediaStream;
+  exports.MediaStreamTrack = window.MediaStreamTrack;
+  exports.RTCDataChannel = window.RTCDataChannel;
+  exports.RTCDataChannelEvent = window.RTCDataChannelEvent;
+  exports.RTCDtlsTransport = window.RTCDtlsTransport;
+  exports.RTCIceCandidate = window.RTCIceCandidate;
+  exports.RTCIceTransport = window.RTCIceTransport;
+  exports.RTCPeerConnection = window.RTCPeerConnection;
+  exports.RTCPeerConnectionIceEvent = window.RTCPeerConnectionIceEvent;
+  exports.RTCRtpReceiver = window.RTCRtpReceiver;
+  exports.RTCRtpSender = window.RTCRtpSender;
+  exports.RTCRtpTransceiver = window.RTCRtpTransceiver;
+  exports.RTCSctpTransport = window.RTCSctpTransport;
+  exports.RTCSessionDescription = window.RTCSessionDescription;
+  exports.getUserMedia = window.getUserMedia;
+  exports.mediaDevices = navigator.mediaDevices;
+}