diff --git a/src/executors/build/build.impl.js b/src/executors/build/build.impl.js
index 65a222d798d60f32dffe0736dc8692abfd85ccdd..460ab35b43e99e0d170d914c21906695ccb88340 100644
--- a/src/executors/build/build.impl.js
+++ b/src/executors/build/build.impl.js
@@ -7,27 +7,8 @@ const _extends = require("@swc/helpers/lib/_extends.js").default;
 require("dotenv/config");
 const _vite = require("vite");
 const _optionsUtils = require("../../utils/options-utils");
-const _js = require("@nrwl/js");
-const _fs = require("fs");
-const _path = require("path");
 async function viteBuildExecutor(options, context) {
-    const projectRoot = context.workspace.projects[context.projectName].root;
     await runInstance(await (0, _optionsUtils.getBuildAndSharedConfig)(options, context));
-    const libraryPackageJson = (0, _path.resolve)(projectRoot, 'package.json');
-    const rootPackageJson = (0, _path.resolve)(context.root, 'package.json');
-    // For buildable libs, copy package.json if it exists.
-    if ((0, _fs.existsSync)(libraryPackageJson) && rootPackageJson !== libraryPackageJson) {
-        await (0, _js.copyAssets)({
-            outputPath: options.outputPath,
-            assets: [
-                {
-                    input: '.',
-                    output: '.',
-                    glob: 'package.json'
-                }
-            ]
-        }, context);
-    }
     return {
         success: true
     };