diff --git a/src/executors/utils/nx-project-paths/index.js b/src/executors/utils/nx-project-paths/index.js
index 929139232bf12d84b4b6e04523684efd012b5fa7..c4ca5dd47d90fcac3727a4799729e396e9ae95e9 100644
--- a/src/executors/utils/nx-project-paths/index.js
+++ b/src/executors/utils/nx-project-paths/index.js
@@ -14,7 +14,7 @@ exports.default = (opts) => {
         name: 'nx-project-paths',
         enforce: 'pre',
         configResolved(config) {
-            const tsConfigBaseJson = (0, path_1.resolve)(config.root, (0, path_1.join)((0, vite_1.normalizePath)(opts.workspaceRoot), 'tsconfig.base.json'));
+            const tsConfigBaseJson = (0, path_1.resolve)(config.root, (0, path_1.join)((0, vite_1.normalizePath)(opts.workspaceRoot), 'tsconfig.json'));
             const extensions = getFileExtensions(opts.extensions);
             resolver = createResolver(tsConfigBaseJson, extensions);
         },
diff --git a/src/generators/library/library.js b/src/generators/library/library.js
index 888cedf5c7a6f0cbd61016a022c37f68f6b5d6c2..a5d8d742ce4451c8c4730ba10ec8ec3865b15f42 100644
--- a/src/generators/library/library.js
+++ b/src/generators/library/library.js
@@ -95,7 +95,7 @@ function addProject(host, options) {
     }, options.standaloneConfig);
 }
 function updateBaseTsConfig(host, options) {
-    (0, devkit_1.updateJson)(host, 'tsconfig.base.json', (json) => {
+    (0, devkit_1.updateJson)(host, 'tsconfig.json', (json) => {
         const c = json.compilerOptions;
         c.paths = c.paths || {};
         delete c.paths[options.name];