diff --git a/index.d.ts b/index.d.ts
index 4afd09d513a5b5217b558f9ea06bf1f8a0d41011..daa90b319eceb420f753a045547ffc5fd0c17653 100755
--- a/index.d.ts
+++ b/index.d.ts
@@ -691,3 +691,5 @@ export function utcTicks(start: Date, stop: Date, count: number): Date[];
  * If there is no associated interval, such as when start or stop is invalid, returns null.
  */
 export function utcTickInterval(start: Date, stop: Date, count: number): TimeInterval | null;
+
+export const unixDay: any;
