diff --git a/dist/lib/output/renderer.js b/dist/lib/output/renderer.js
index 86e5e702b2c556324d25544cbf282b242e95800c..a6fbf269b3073c5e397aeef20d6e80263281ecbb 100644
--- a/dist/lib/output/renderer.js
+++ b/dist/lib/output/renderer.js
@@ -126,7 +126,7 @@ let Renderer = class Renderer extends component_1.ChildableComponent {
             return;
         }
         const output = new events_1.RendererEvent(events_1.RendererEvent.BEGIN, outputDirectory, project);
-        output.urls = this.theme.getUrls(project);
+        output.urls = await this.theme.getUrls(project);
         this.trigger(output);
         if (!output.isDefaultPrevented) {
             output.urls.forEach((mapping) => {
diff --git a/dist/lib/output/theme.d.ts b/dist/lib/output/theme.d.ts
index 23d3bed93daa244e5c971eaa420a81a2b023e6cb..a7416dbc20feb2e59cd58e0c618518cf906b0711 100644
--- a/dist/lib/output/theme.d.ts
+++ b/dist/lib/output/theme.d.ts
@@ -35,7 +35,7 @@ export declare abstract class Theme extends RendererComponent {
      * @returns A list of {@link UrlMapping} instances defining which models
      * should be rendered to which files.
      */
-    abstract getUrls(project: ProjectReflection): UrlMapping[];
+    abstract getUrls(project: ProjectReflection): UrlMapping[] | Promise<UrlMapping[]>;
     /**
      * Renders the provided page to a string, which will be written to disk by the {@link Renderer}
      */