diff --git a/dist/node/chunks/dep-0fc8e132.js b/dist/node/chunks/dep-0fc8e132.js
index 54ffaf0a863fc6776e780e25342f901ff0c3965e..a00e599f2d8dad9e3019699e043951e0912398f8 100644
--- a/dist/node/chunks/dep-0fc8e132.js
+++ b/dist/node/chunks/dep-0fc8e132.js
@@ -34778,7 +34778,7 @@ function tryResolveBrowserMapping(id, importer, options, isFilePath, externalize
         const mapId = isFilePath ? './' + slash$1(path$n.relative(pkg.dir, id)) : id;
         const browserMappedPath = mapWithBrowserField(mapId, pkg.data.browser);
         if (browserMappedPath) {
-            const fsPath = path$n.join(pkg.dir, browserMappedPath);
+            const fsPath = bareImportRE.test(browserMappedPath) ? path$n.join(pkg.dir, '../' + browserMappedPath) : path$n.join(pkg.dir, browserMappedPath);
             if ((res = tryFsResolve(fsPath, options))) {
                 isDebug$4 &&
                     debug$c(`[browser mapped] ${picocolors.exports.cyan(id)} -> ${picocolors.exports.dim(res)}`);