diff --git a/dist/node/chunks/dep-DG6Lorbi.js b/dist/node/chunks/dep-DG6Lorbi.js
index 0baef5ef63979769edd787125363d85e903be4ad..08ee3c8acab53dbab784d1abcc1643742f16c1e7 100644
--- a/dist/node/chunks/dep-DG6Lorbi.js
+++ b/dist/node/chunks/dep-DG6Lorbi.js
@@ -52512,18 +52512,15 @@ Object.defineProperty(${ssrModuleExportsKey}, "default", { enumerable: true, con
     }
   });
   let map = s.generateMap({ hires: "boundary" });
+  map.sources = [path.basename(url)]
+  // needs to use originalCode instead of code
+  // because code might be already transformed even if map is null
+  map.sourcesContent = [originalCode]
   if (inMap && inMap.mappings && "sources" in inMap && inMap.sources.length > 0) {
     map = combineSourcemaps(url, [
-      {
-        ...map,
-        sources: inMap.sources,
-        sourcesContent: inMap.sourcesContent
-      },
+      map,
       inMap
     ]);
-  } else {
-    map.sources = [path$n.basename(url)];
-    map.sourcesContent = [originalCode];
   }
   return {
     code: s.toString(),
