diff --git a/dist/v2/index.js b/dist/v2/index.js
index 3de49207d2f532057999eee39150022044b07d17..bf905a91a9e7d3ec66fe23d39ea5ad53ec7b2269 100644
--- a/dist/v2/index.js
+++ b/dist/v2/index.js
@@ -24,14 +24,6 @@ const typedocPlugin = (opts) => {
         if (!project) {
             return;
         }
-        if (typedocOptions.watch) {
-            app.convertAndWatch(async (project) => {
-                app.generateDocs(project, outputDirectory);
-            });
-        }
-        else {
-            app.generateDocs(project, outputDirectory);
-        }
         return {
             name: 'vuepress-plugin-typedoc',
             extendsPageOptions: (pageOptions) => {